docs/enableexperimental.md

```diff
@@ -0,0 +1,36 @@
+# How to enable experimental features
+
+Fleet supports experimental features that are disabled by default and that can be enabled by the user.
+
+Enabling/disabling experimental feaures is done usign extra environment variales that are available when deploying rancher/fleet.
+
+## Enabling an experimental feature
+
+At the moment we're writing this document fleet has OCI storage as an experimental feature.
+### Enabling when installing fleet stand-alone
+
+All you need to do is to pass something like:
+```
+--set-string extraEnv[0].name=EXPERIMENTAL_OCI_STORAGE \
+--set-string extraEnv[0].value=true \
+```
+to your helm install or update command. 
+
+Please note you have to use `--set-string` because otherwise the boolean value won't work as expected.
+
+### Enabling when installing fleet with rancher
+
+You can also activate the experimental features in `Fleet` when installing `Rancher`.
```

Comment:
I think it would be interesting to discuss the use of upper- vs lowercase first letter in naming, ie `[Ff]leet` and `[Rr]ancher`, as I'd do the opposite here:
(rationale: using uppercase when referring to products/programs, but lowercase when referring to chart names as they'd appear on the command line; not sure if that's shared practice though, hence the interest in a (brief) discussion)
```suggestion
### Enabling when installing Fleet with Rancher

You can also activate the experimental features in `fleet` when installing `rancher`.
```